cmds/rush/parse.go

```diff
@@ -62,7 +63,7 @@ func one(b *bufio.Reader) byte {
 		return 0
 	}
 	if err != nil {
-		panic(fmt.Errorf("reading bufio: %v", err))
+		log.Fatalf("reading bufio: %v", err)
```

Comment:
It doesn't fix the problem. What's going on is a process is vanishing from the set of processes 
 that make up the Go program. The Go runtime doesn't handle that real well. The only problem this fixes is the resulting infinite set of messages about reading bufio.  The big problem is why the heck one process is vanishing, but not all of them; but this at least makes the mess bearable.